module/os/linux/zfs/zfs_vfsops.c

```diff
@@ -2126,13 +2135,14 @@ zfs_get_vfs_flag_unmounted(objset_t *os)
 	return (unmounted);
 }
 
-void
+int
```

Comment:
I expect these changes also need to be added on the ZFS side.

Reply:
Not sure what specifically you mean here? I think the change you marked up is updating the return of zfs_init from a void to an int, but this file is specific to the linux kernel module so I'm not sure what would need to be changed on the freebsd side to account for it?

Thanks...

Reply:
zfs_init is called from common code (see your change to zfs_ioctl.c). Each platform has its own implementation of this function. FreeBSD's is in module/os/freebsd/zfs/zfs_vfsops.c

Reply:
Oh, ok, I see. Updated. Any other things that need to be reflected in the freebsd side?